custom_components/mqtt_vacuum_camera/camera.py

```
@@ -455,85 +367,33 @@ async def _process_parsed_json(self, test_mode: bool = False):
-        self._attr_frame_interval = max(0.1, total_time)
-
-        LOGGER.debug(
-            "%s: Image Completed - Total image processing time (receive to display): %.3fs",
-            self._file_name,
-            total_time,
-        )
-
-    async def async_pil_to_bytes(
-        self, pil_img, image_id: str = None
-    ) -> Optional[bytes]:
+    def pil_to_bytes(self, pil_img, image_id: str = None) -> Optional[bytes]:
```

Ruff (0.14.0)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)